packages/storybook/stories/storybook.test.ts

```diff
@@ -15,15 +15,18 @@ jest.mock('@azure/communication-calling', () => {
 
 ReactDom.createPortal = (node: any) => node;
 
-// Classnames in fluent are of the format css-#, where # is an integer that is incremented by a global singleton
+// Classnames in fluent are of the format <display name>-#, where # is an integer that is incremented by a global singleton
```

Comment:
Sorry, I don't understand this. The display name of what?

Reply:
Just what fluent calls it: https://github.com/microsoft/fluentui/blob/66dc6e9684baf35e7930622e1f3accdb51ba8516/packages/merge-styles/src/Stylesheet.ts#L44
I'll update this to `<classname-prefix>-#` and add an example for clarity!

Reply:
updated